uploadNeuroDB/NeuroDB/MRI.pm

```diff
@@ -208,41 +216,49 @@ sub getSessionID {
                 }
             }
         }
-
+    
         $newVisitNo = 1 unless $newVisitNo;
         $centerID = 0 unless $centerID;
 
-#### insert the new session setting Current_stage to 'Not started' because that column is important to the behavioural data entry gui.
-	$query = "INSERT INTO session SET CandID=".$dbh->quote($subjectIDref->{'CandID'}).", Visit_label=".$dbh->quote($subjectIDref->{'visitLabel'}).", CenterID=$centerID, VisitNo=$newVisitNo, Current_stage='Not Started', Scan_done='Y', Submitted='N', SubprojectID=".$dbh->quote($objective);
- 	$dbh->do($query); # execute query
-	$sessionID = $dbh->{'mysql_insertid'}; # retain id of inserted row
-	$subjectIDref->{'visitNo'} = $newVisitNo; # add visit number to subjectIDref
-
-	# check dates of other files
-	if(defined($studyDateJD)) {
-	    # get the set of sessions for the subject
-	    $query = "SELECT ID FROM session WHERE CandID=$subjectIDref->{'CandID'} AND Active='Y'";
-	    $sth = $dbh->prepare($query);
-	    $sth->execute();
-
-	    if($sth->rows > 0) {
-		my @sessionIDs = ();
-		while(my $session = $sth->fetchrow_array()) { push @sessionIDs, $session[0]; }
-		$sth->finish();
-
-		# get the set of files
-		$query = "SELECT FileID FROM files WHERE SessionID IN (".join(',', @sessionIDs)." AND FileType='mnc' AND OutputType='native'";
-		$sth = $dbh->prepare($query);
-		$sth->execute();
-
-		if($sth->rows > 0) {
-		    my @files = ();
-		    while(my $filearray = $sth->fetchrow_array()) { push @files, $filearray[0]; }
-
-		    $sth->finish();
-		} # end if sth->rows (files)
-	    } # end if sth->rows (sessionIDs)
-	} # end if defined studyDateJD
+        #### insert the new session setting Current_stage to 'Not started' because that column is important to the behavioural data entry gui.
+        $query = "INSERT INTO session "
+               . "SET CandID=".$dbh->quote($subjectIDref->{'CandID'}).", "
+               . "    Visit_label=".$dbh->quote($subjectIDref->{'visitLabel'}).", "
+               . "    CenterID=$centerID, "
+               . "    VisitNo=$newVisitNo, "
+               . "    Current_stage='Not Started', "
+               . "    Scan_done='Y', "
+               . "    Submitted='N', "
+               . "    SubprojectID=".$dbh->quote($subjectIDref->{'SubprojectID'});
```

Comment:
shouldn't this contain the ProjectID too?

Reply:
See previous comments.

Reply:
I am guessing you referred to your comments saying there is no ProjectID in session table? ProjectID is in the session table and should also be filled here. See previous comment on the topic